libs/elements/cards/src/lib/components/elewa-price-list/elewa-price-list.component.ts

```diff
@@ -0,0 +1,19 @@
+import { Component, Input } from '@angular/core';
+import { __highlightedPricingList } from '@elewa-website/data/sections';
+
+import { PriceItem } from '@elewa-website/models/schema/ui/cards';
+
+
+@Component({
+  selector: 'elewa-website-elewa-price-list',
+  templateUrl: './elewa-price-list.component.html',
+  styleUrls: ['./elewa-price-list.component.scss'],
+})
+export class ElewaPriceListComponent {
+  /** 
+  * This variable represents a list of PriceItem objects
+  * that are highlighted or featured in a pricing list.
+ */
+  @Input() highlightedPricingList: PriceItem[] = __highlightedPricingList;
+
```

Comment:
Since this is a reusable component there's no need to initialise with the mockData. whatever component reuses this component will pass the mockData.